wasted-app/RestaurantViewFoodControl.cs

```diff
@@ -33,9 +34,25 @@ private void ListRestaurantFoodItems()
             }
         }
 
+
         private void BackButton_Click(object sender, EventArgs e)
         {
             MainForm.mainForm.panel.Controls.Remove(this);
         }
+
+        private void SortComboBox_SelectedIndexChanged(object sender, EventArgs e)
+        {
+            foodPanel.Controls.Clear();
+            GetRestaurantFoodItems();
+            if (SortComboBox.SelectedIndex == 0)
+            {
+                Foods = Foods.SortByPrice();
```

Comment:
maybe move `Foods = Foods.SortByPrice()` and `SortByNew` to separate small functions?

Reply:
I had that idea too. I can change it